morphir/runtime/src/org/finos/morphir/runtime/quick/testing/UnitTesting.scala

```diff
@@ -31,6 +31,10 @@ object UnitTesting {
   def expectationType = T.reference("Morphir.UnitTest", "Expect", "Expectation")
   def testPrefix      = "Morphir.UnitTest:Test:"
   def expectPrefix    = "Morphir.UnitTest:Expect:"
+  def testPackagePath = fqn"Morphir.UnitTest:Test:foo".packagePath
+  def containsTestCode(globals: GlobalDefs, fqn: FQName, definition: TypedDefinition): Boolean =
```

Comment:
Now that we have the `getStaticallyReachable` function, we should use that here as well - if any reachable function comes from the test package, the function itself should be treated as a test function. This will exclude test-based helper functions from coverage reporting.